build/pdp10-kl/config.202

```diff
@@ -515,9 +515,10 @@ DEFOPT IMPUS==106	; ARPA net host number
 DEFOPT IMPUS3==<IPADDR 192,168,1,100>	; Internet host number
 DEFOPT NCPP==0		; Flush NCP code
 DEFOPT NNETCH==30.	; # NCP network channels
-DEFOPT CHAOSP==0	;NO CHAOS NET
-DEFOPT MYCHAD==1440	;CHAOS NET ADDRESS
+DEFOPT CHAOSP==1	;NO CHAOS NET
+DEFOPT MYCHAD==177002	;CHAOS NET ADDRESS
```

Comment:
Ditto.

Reply:
(At least on klh10, this address is overridden by the devdef address.)